manifests/s/SnapmakerDevTeam/SnapmakerLuban/4.1.1/SnapmakerDevTeam.SnapmakerLuban.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: SnapmakerDevTeam.SnapmakerLuban
+PackageVersion: 4.1.1
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
```

Comment:
```suggestion
- Architecture: x86
```

Reply:
Oops, didn't see this.

ia32 is 32-bit

cc @Trenly

Reply:
Thanks, I missed that too 🤣